package.json

```diff
@@ -65,7 +65,7 @@
         "onCommand:appService.RevealConnectionInAppSettings",
         "onView:azureAppService"
     ],
-    "main": "./out/src/extension",
+    "main": "./entrypoint.js",
```

Comment:
I think we should discuss the entrypoint/main/extension naming. Here's my suggestions:
1. entrypoint.js -> main.js (to match the package.json name here)
1. Don't change the name of extension.ts (so we don't have to learn a new name)
1. Make webpack generate a `webpackedExtension.js` file instead of `extension.js` file

If this is the current file structure:
- dist
    - extension.js
- src
    - main.ts
- entrypoint.js
- extension.ts

I think my suggestion would result in a file structure like this:
- dist
    - webpackedExtension.js
- src
    - extension.ts
- main.js
- webpackedExtension.ts

Reply:
How about this:

- dist
    - main.bundle.js
- src
    - extension.ts
- main.js
- main.bundle.ts

Reply:
I like the `.bundle` pattern, but shouldn't it be `extension.bundle` instead of `main.bundle`?
- dist
    - extension.bundle.js
- src
    - extension.ts
- main.js
- extension.bundle.ts

Webpack doesn't bundle `main.js` - it bundles `extension.ts`, right? Also it just feels weird to me that `main.js` would import `./dist/main.bundle.js` (Feels like that implies it's importing a bundle of itself)

Reply:
Works for me.